tests/ModelTest.php

```diff
@@ -670,6 +673,30 @@ public function testUnsetDotAttributesAndSet(): void
         $this->assertSame(['note2' => 'DEF', 'note1' => 'ABC'], $user->notes);
     }
 
+    public function testDateUseLocalTimeZone(): void
+    {
+        $tz = 'Australia/Sydney';
+        date_default_timezone_set($tz);
```

Comment:
This test permanently alters the state of the environment. Would it be preferable to capture the current timezone first and then restore it at the end of the test?

If you're concerned about a failure interrupting things, you can use `tearDown()`, but I don't think that's necessary.

Reply:
The timezone is reset before each test by this: https://github.com/orchestral/testbench-core/blob/bf3de053bc6566210c0c336635d03e97e270f205/src/Concerns/CreatesApplication.php#L389-L391

But I can explicitly reset the timezone to prevent any regression.

Reply:
Ah, I didn't realize the tests in this project used OrchestraTestCase instead of PHPUnit directly. I think a comment explaining that would be sufficient.